sdkconfig

```diff
@@ -164,10 +167,10 @@ CONFIG_ESPTOOLPY_FLASHFREQ_20M=
 CONFIG_ESPTOOLPY_FLASHFREQ="80m"
 CONFIG_ESPTOOLPY_FLASHSIZE_1MB=
 CONFIG_ESPTOOLPY_FLASHSIZE_2MB=
-CONFIG_ESPTOOLPY_FLASHSIZE_4MB=
+CONFIG_ESPTOOLPY_FLASHSIZE_4MB=y
 CONFIG_ESPTOOLPY_FLASHSIZE_8MB=
-CONFIG_ESPTOOLPY_FLASHSIZE_16MB=y
-CONFIG_ESPTOOLPY_FLASHSIZE="16MB"
+CONFIG_ESPTOOLPY_FLASHSIZE_16MB=
+CONFIG_ESPTOOLPY_FLASHSIZE="4MB"
```

Comment:
Partition map is 4MB so I guess this 16MB setting never mattered?

Reply:
This seemed odd to me. I double checked the settings from the official M5Stack repo, 4MB (https://github.com/m5stack/M5Stack-IDF/blob/9b803b4adede63f70722a42b1b6297b2bc63ae60/sdkconfig#L80). Must not have mattered before.